src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -10,12 +10,11 @@
 import static io.vavr.control.Validation.invalid;
 import static io.vavr.control.Validation.valid;
 import static java.lang.String.format;
-import static uk.gov.hmcts.reform.ccd.client.model.CallbackTypes.ABOUT_TO_SUBMIT;
 
-interface CallbackValidations {
-    Logger log = LoggerFactory.getLogger(CallbackProcessor.class);
+public interface CallbackValidations {
+    Logger log = LoggerFactory.getLogger(CallbackValidations.class);
     String ATTACH_TO_CASE_REFERENCE = "attachToCaseReference";
-    String ATTACH_RECORD = "attachRecord";
+    String ATTACH_TO_EXISTING_CASE = "attachToExistingCase";
```

Comment:
constant doesn't explain what this is.
it's like `FOO = "foo"`

Reply:
does it actually need to ... its either that or leave it inline.
There really isn't any better naming unless you ATTACH_TO_EXISTING_CASE_TYPE ... which really does not add much except verbosity.